docs/install/requirements.md

```diff
@@ -0,0 +1,51 @@
+RKE2 is very lightweight, but has some minimum requirements as outlined below.
+
+## Prerequisites
+
+Two nodes cannot have the same hostname.
+
+If all your nodes have the same hostname, use `--node-name` or `$RKE2_NODE_NAME` for each node you add to the cluster to have a different node name.
+
+## Operating Systems
+
+RKE2 is officially supported and tested on the following operating systems and their subsequent non-major releases:
+
+*    Ubuntu 18.04 (amd64)
+*    Ubuntu 20.04 (amd64)
+*    CentOS/RHEL 7.8 (amd64)
+*    CentOS/RHEL 8.x (amd64)
+
+## Hardware
+
+Hardware requirements scale based on the size of your deployments. Minimum recommendations are outlined here.
+
+*    RAM: 512MB Minimum (we recommend at least 1GB)
+*    CPU: 1 Minimum
+
+#### Disks
+
+RKE2 performance depends on the performance of the database, and since RKE2 runs etcd embeddedly and it stores the data dir on disk. we recommend using an SSD when possible to ensure optimal performance.
```

Comment:
```suggestion
RKE2 performance depends on the performance of the database, and since RKE2 runs etcd embeddedly and it stores the data dir on disk, we recommend using an SSD when possible to ensure optimal performance.
```